config/configgrpc/configgrpc.go

```diff
@@ -360,6 +372,20 @@ func (gss *ServerConfig) toServerOption(host component.Host, settings component.
 		})
 	}
 
+	if gss.MemoryLimiter != nil {
+		memoryLimiter, err := getMemoryLimiterExtension(gss.MemoryLimiter, host.GetExtensions())
+		if err != nil {
+			return nil, err
+		}
+
+		uInterceptors = append(uInterceptors, func(ctx context.Context, req any, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (resp any, err error) {
+			return memoryLimiterUnaryServerInterceptor(ctx, req, info, handler, memoryLimiter)
+		})
+		sInterceptors = append(sInterceptors, func(srv any, ss grpc.ServerStream, info *grpc.StreamServerInfo, handler grpc.StreamHandler) error {
+			return memoryLimiterStreamServerInterceptor(srv, ss, info, handler, memoryLimiter)
```

Comment:
This path would require setting up streaming server/client which doesn't seem necessary, as I already have coverage for `memoryLimiterStreamServerInterceptor`.